soccer/Robot.hpp

```diff
@@ -4,30 +4,30 @@
 #include <planning/CompositePath.hpp>
 #include <planning/InterpolatedPath.hpp>
 #include <planning/MotionCommand.hpp>
-#include <planning/RobotConstraints.hpp>
 #include <planning/RRTPlanner.hpp>
-#include "planning/RotationCommand.hpp"
+#include <planning/RobotConstraints.hpp>
 #include "planning/DynamicObstacle.hpp"
+#include "planning/RotationCommand.hpp"
 
+#include <protobuf/Control.pb.h>
 #include <protobuf/RadioRx.pb.h>
 #include <protobuf/RadioTx.pb.h>
-#include <protobuf/Control.pb.h>
 #include <Utils.hpp>
 
+#include <stdint.h>
+#include <Eigen/Dense>
```

Comment:
This stuff should probably be in the cpp if possible

Reply:
There's a function returning Eigen stuff so it should be in the header, same with stdint.h, there's a function using uint8_t